app/schemas.py

```diff
@@ -289,7 +301,20 @@ class Meta:
             'sms_sender',
             'permissions',
             'inbound_number',
-            'inbound_sms'
+            'inbound_sms',
+            'all_template_folders',
+            'annual_billing',
+            'contact_list',
+            'created_by',
+            'crown',
+            'inbound_number',
```

Comment:
inbound_number and inbound_sms are repeated in this list

Reply:
Corrected and squashed out